pkg/expapi/types.go

```diff
@@ -361,3 +361,38 @@ type ThirdPartyResourceDataList struct {
 	// Items is a list of third party objects
 	Items []ThirdPartyResourceData `json:"items"`
 }
+
+// LockSpec is the specification of a resource lock.
+type LockSpec struct {
+	// Who currently has the lock.
```

Comment:
Please make these valid godoc comments. The should start with the name of the thing they describe.